### ISSUE TITLE
Advice on exposing host daemon

### ISSUE THREAD

**Issue by razuro** [slonopotamus/stevedore#63]
**TLDR;**
Is there a substitute for mounting `/var/run/docker.sock:/var/run/docker.sock` in docker-compose file?

---
**Long Version**
I'm fully aware that Linux container is not supported. Hence, for my use case I'm relying on v0.9.0 and managed to get what is needed to work, except communicating to host docker daemon. This is merely an advice seeking/hoping someone who got it working can share what they did.

The great thing about stevedore that is not highlighted in the README, is that it supports WSL version 1, which none of the alternatives are: (Rancher Desktop, containerd, Podman). This is a great feature for those who are working with Windows Server 2019 that doesn't support WSL2.

What I've known/tried:
1. [Stevedore did not expose TCP](https://github.com/slonopotamus/stevedore/issues/46)
2. There is a way, by using [wsl-proxy](https://github.com/slonopotamus/stevedore/issues/8)

TIA 🙏🏻

**Comment by slonopotamus** [comment]
Stevedore doesn't support Linux containers anymore. There are plenty of other FOSS projectd that do, use any of them.
